### PR TITLE
Fixed warning message style and api key issues.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* /bcgov/entity###

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/App.vue [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
-    <v-alert
-      tile dense
-      type="warning"
-      v-if="bannerText">
-      <div v-html="bannerText" class="mb-0 text-center colour-dk-text"></div>
-    </v-alert>
-
     <div class="app-body">
       <main>
+        <sbc-system-banner
```
Comment (pwei1018) [inline ppr-ui/src/App.vue:25]: I can't make v-alert works in ppr-ui. I bring back the sbc-system-banner.

Comment (cameron-eyds) [reply ppr-ui/src/App.vue:25]: It could be our Vuetify version.

Comment (pwei1018) [reply ppr-ui/src/App.vue:25]: Ok, I will leave it. When you upgrade to VUE3, you may review this.

**Review by cameron-eyds — APPROVED** [review]
LG!

File: ppr-ui/src/App.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/App.vue:25]: Definitely 👍

File: ppr-ui/src/utils/feature-flags.ts [hunk 2]
```diff
@@ -10,7 +10,9 @@ export const defaultFlagSet: LDFlagSet = {
   'search-registration-number': false,
   'search-serial-number': false,
   'mhr-ui-enabled': false,
-  'mhr-registration-enabled': false
+  'mhr-registration-enabled': false,
+  'sentry-enable': false, // by default, no sentry logs
```
Comment (dimak1) [inline ppr-ui/src/utils/feature-flags.ts:14]: Nice addition 👍🏻

**Review by dimak1 — APPROVED** [review]

**Pull request merged**